app/decorators/models/super_good/solidus_taxjar/spree/state_decorator.rb

```diff
@@ -0,0 +1,14 @@
+module SuperGood
```

Comment:
Could you add a commit body for this commit explaining a little bit more about exempt regions and the change motivation?

Reply:
Sure, i'll add that